package.json

```
@@ -79,24 +81,25 @@
     "webpack": "^4.33.0"
   },
   "scripts": {
+    "a11y-test": "yarn build-storybook && jest --testPathIgnorePatterns packages storybook",
     "bootstrap": "lerna bootstrap --use-workspaces",
-    "build:cjs": "lerna exec --parallel -- babel --extensions '.js,.ts,.tsx' --ignore **/*.story.js,**/*.story.ts,**/*.spec.js,**/*.spec.ts,**/*.test.js,**/*.test.ts,**/__tests__/** --root-mode upward --source-maps true src -d lib",
-    "build:esm": "lerna exec --parallel -- cross-env BABEL_ENV=esm babel --extensions '.js,.ts,.tsx' --ignore **/*.story.js,**/*.story.ts,**/*.spec.js,**/*.spec.ts,**/*.test.js,**/*.test.ts,**/__tests__/** --root-mode upward --source-maps true src -d esm",
+    "build:cjs": "lerna exec --parallel -- babel --extensions '.js,.ts,.tsx' --ignore **/*.story.js,**/*.story.ts,**/*.spec.js,**/*.spec.ts,**/*.test.js,**/*.test.ts,**/__tests__/**,storybook-static/**/* --root-mode upward --source-maps true src -d lib",
+    "build:esm": "lerna exec --parallel -- cross-env BABEL_ENV=esm babel --extensions '.js,.ts,.tsx' --ignore **/*.story.js,**/*.story.ts,**/*.spec.js,**/*.spec.ts,**/*.test.js,**/*.test.ts,**/__tests__/**,storybook-static/**/* --root-mode upward --source-maps true src -d esm",
     "prepublish": "yarn typescript && yarn build:cjs && yarn build:esm",
     "check-eslint-config": "eslint --print-config jestconfig.js | eslint-config-prettier-check",
     "coverage": "jest --coverage",
     "deploy-storybook": "storybook-to-ghpages",
     "dev": "start-storybook -p 5555",
     "predev": "yarn",
-    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --quiet",
-    "lint:fixjs": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --fix",
+    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --ignore-pattern storybook-static --quiet",
+    "lint:fixjs": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-pattern esm --ignore-pattern lib --ignore-pattern node_modules --ignore-pattern storybook-static --fix",
     "lint:styles": "stylelint \"packages/**/!(*.d).{j,t}s{,x}\"",
     "lint": "yarn lint:js && yarn lint:styles",
     "prettier": "prettier --write \"**/*.{json,md,yml}\"",
     "semantic-release": "lerna exec --concurrency 1 -- semantic-release -e semantic-release-monorepo",
     "test": "yarn lint:js && yarn lint:styles && yarn typescript && yarn unit",
```

Please add the `a11y-test` to this.

```suggestion
    "test": "yarn lint:js && yarn lint:styles && yarn typescript && yarn unit && yarn a11y-test",
```

---

Added in fd2a17e112b55b92618d006ebcd5730b290629e3